app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -100,7 +100,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsNothing = false
         allProducts!!.forEach {
-            if (!it.product.equals("Human Remains")) {
+            if (it.product != "Human Remains") {
```

Comment:
simplified operator

Reply:
This was covered in a previously submitted review. When that occurs, first in wins.